src/components/utils/useLayer.ts

```diff
@@ -28,8 +31,13 @@ export function useLayer({
         onOutsideClick,
         onClose,
         contentRefs,
+        idle,
```

Comment:
If I understood you correctly, changes of `idle` are not handled right now (and personally I wouldn't like `idle` to have a dynamic value), changes of `open` and `enabled` are handled since they are dependencies of `useEffect`: https://github.com/gravity-ui/uikit/pull/492/files/db588767a7cb9d6b1668a51c743c99edb8f914b7#diff-d28d4a02c53c486a096624c2ebcb7af8be0a1fecc256edaf8f226cd2aadaa016L66